projects/components/src/data-exporter/data-exporter.component.ts

```diff
@@ -72,22 +73,26 @@ export class DataExporterComponent implements OnInit {
     /**
      * Text for the Dialog Header
      */
-    @Input() dialogHeader: string;
+    @Input()
+    dialogHeader: TranslatedText = this.translationService.translateAsync('vcd.cc.data-exporter.title');
```

Comment:
We can create a mixin called `CanTranslate` that can allow our components to have a `translate()` method directly through delegation, saving typing everywhere.

See https://codesandbox.io/s/mixins-with-abstract-properties-eh9ce?file=/index.ts:123-394